src/dispatch/alembic/versions/5d4dee3e24fc_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 5d4dee3e24fc
+Revises: 8b67c774279d
+Create Date: 2020-03-27 10:47:57.672426
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5d4dee3e24fc'
+down_revision = '8b67c774279d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('conference', sa.Column('conference_challenge', sa.String(), server_default='N/A', nullable=False))
```

Comment:
Why not let this be null? Null will return `None` which can be checked more easily that the string value of "N/A"

Reply:
when this was Null, there are errors on the loading up the incident table in the UI, so I set this to "N/A" instead